lib/takeaway.rb

```diff
@@ -0,0 +1,43 @@
+require './lib/menu.rb'
+require './lib/sms_sender.rb'
+class Takeaway
+
+attr_reader :customer_order
+
+ def view_menu
+  Menu.new.menu
+ end
+
+ def place_order(order_hash, exact_total)
+  @customer_order = order_hash
+  order_checker
+  message = 'sorry your calculated sum does not match the order total'
+  raise(message) if order_sum != exact_total
```

Comment:
Style/SignalException: Use `fail` instead of `raise` to signal exceptions. (https://github.com/bbatsov/ruby-style-guide#fail-method)